### PR TITLE
Exalt General Proposal of Rebalance

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
I toned the amount of money the Exalt receive by a little by making them receiving a little more money but not outright removing the debtor perk.

The organs have extra modifiers to make them more efficient and somehow others organs have better efficiency

Gives exalts a new passive perk that gives them more insight but not being better than a mar'quan and not worse than the neurolink

Exalts can have 3 languages

Pending Lore-Wise: Exalts can be 520 years old.

Exalts gets more hungry than normal

Buffs exalts stats

Gives Exalts more darksight

Exalts get 20+ extra health

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by ActuallyALovelyGrave** [comment]
So, basically they will have 4 languages total(considering this buff + Diverse Culture)?
Also, was overall `nutriment_req = 40` tested out? Because it might result into being *always* hungry with no end, and that isn't fun.
*Notices `hunger_factor = 2.2*
Oh god, this will likely be *always hungry*

File: code/modules/mob/living/carbon/human/species/station/station.dm [hunk 1]
```diff
@@ -47,27 +47,29 @@
...
 	especially handsome humans, but anything is possible with enough splicing."
-	num_alternate_languages = 2
+	num_alternate_languages = 3 //Extra language due to how long they live and how fast learners they are in general.
 	name_language = null // Use the first-name last-name generator rather than a language scrambler
 	min_age = 18
-	max_age = 200 //Effectively ageless, but human history is only so long.
-	hunger_factor = 1.1
+	max_age = 520 //Effectively ageless, but human history is only so long. Abhuman Adaptation was on 2139 and such was the Exalt creations AFAIK.
+	hunger_factor = 2.2 // Way bigger hunger due to their organs. Well you could say that their bodies has better efficiency in using nutrients but someone with a bigger brain and stronger muscles need more mass and nutrients.
+	taste_sensitivity = TASTE_HYPERSENSITIVE
```
Comment (ActuallyALovelyGrave) [inline code/modules/mob/living/carbon/human/species/station/station.dm:55]: Undocumented change?

**Comment by Guidesu** [comment]
> So, basically they will have 4 languages total(considering this buff + Diverse Culture)? Also, was overall `nutriment_req = 40` tested out? Because it might result into being _always_ hungry with no end, and that isn't fun. _Notices `hunger_factor = 2.2_ Oh god, this will likely be _always hungry_

Nutrient Req AFAIK is how much a organ needs to have to function

**Comment by ActuallyALovelyGrave** [comment]
> > So, basically they will have 4 languages total(considering this buff + Diverse Culture)? Also, was overall `nutriment_req = 40` tested out? Because it might result into being _always_ hungry with no end, and that isn't fun. _Notices `hunger_factor = 2.2_ Oh god, this will likely be _always hungry_
> 
> Nutrient Req AFAIK is how much a organ needs to have to function

Fair point since rn not even im sure about this

**Comment by Trilbyspaceclone** [comment]
Exalts are in a good place as is not seeing a value to any of these changes

**Pull request closed without merging**